src/edu/washington/escience/myria/parallel/Server.java

```diff
@@ -913,8 +919,103 @@ public DatasetStatus ingestDataset(
   }
 
   /**
-   * @param relationKey the relationalKey of the dataset to add
-   * @param schema the schema of the dataset to add
+   * Parallel Ingest
+   *
+   * @param relationKey the name of the dataset.
+   * @param workersToIngest restrict the workers to ingest data (null for all)
+   * @throws URIException
+   * @throws DbException
+   * @throws InterruptedException
+   */
+  public DatasetStatus parallelIngestDataset(
```

Comment:
So for now parallel ingest is only available through a new REST API? What is the new REST endpoint corresponding to this server method? How will we expose it from the MyriaL `IMPORT` statement--will we just sniff the URI for the S3 protocol?